docs/content/reference/SLOs.md

```diff
@@ -0,0 +1,23 @@
+# SLOs
+This project is committed to satisfy a number of internal SLOs. 
+These SLOs can be taken as reference by consumers to aggregate them and help define their own SLOs.
+
+These SLOs/SLIs are currently just referential and monitored as part of our CI runs.
+They are valid under the following circumstances:
+
+- Running raw Hypershift in a regular OCP management cluster. I.e. Just Hypershift operator and bare HostedCluster and NodePool resources.
```

Comment:
What is meant by 'regular' ? If it means 'standalone' I think that it's too limiting. We should be able to satisfy SLOs running on a hosted control plane OCP as well.